modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/TeamMemberInputs/index.jsx

```diff
@@ -23,11 +28,42 @@ const TeamMemberInputs = ({
 	placeholderEmail,
 	selectOnChange,
 }) => {
+	const provisioningService = useProvisioningLicenseKeys();
+
 	const bannedDomains = useBannedDomains(
 		invite?.email,
 		FETCH_DELAY_AFTER_TYPING
 	);
 
+	const extractDomain = (email) => {
+		const domain = email.split('@')[1];
+
+		return domain;
+	};
```

Comment:
Function not needed, also it could be placed outside the component to avoid re-rendering